src/test/java/bridge/BridgeMakerTest.java

```diff
@@ -0,0 +1,45 @@
+package bridge;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+import java.util.Arrays;
+import java.util.List;
+import org.assertj.core.api.Assertions;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+class BridgeMakerTest {
+
+    BridgeMaker upBridgeMaker = new BridgeMaker(new BridgeNumberGenerator() {
+        @Override
+        public int generate() {
+            return 1;
+        }
+    });
```

Comment:
```suggestion
    BridgeMaker upBridgeMaker = new BridgeMaker(() -> 1);
```

`BridgeNumberGenerator` 는 함수형 인터페이스이므로 간단하게 작성할 수 있습니다. 익명 클래스를 사용하신 이유가 있을까요?